app/views/pages/_form.html.erb

```diff
@@ -11,24 +11,49 @@
 
   <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
 
-  <%= f.govuk_check_boxes_fieldset :is_optional, multiple: false, legend: { text: t("pages.add_rules_legend") , size: 'm' } do %>
-    <%= f.govuk_check_box :is_optional, true,0, multiple: false,
-                          small: true,
-                          label: { text: t("pages.make_optional_text")  },
-                          hint: { text: t("pages.make_optional_hint") } %>
-
+  <% if page_object.answer_type == 'selection' %>
+    <%= f.hidden_field :is_optional %>
```

Comment:
Why is this a hidden field? 
Wouldn't we want to reset the value if they user edits an existing question (that is set to a type that allows it to be optional) and changes the type to "selection"  and therefore might not be able to mark it optional?

Reply:
@aliuk2012 `is_optional` behaves slightly differently for the checkbox and radio inputs, we're using it as the way to decide whether the 'none of the above' option appears in the runner. So for this input we're setting the value on the selections_settings view, where it's called `Include an option for ‘None of the above’` to make it clearer what it does. 

So for a user editing an existing question, the flow would be:
- go into the edit question page
- click 'Change' on answer type
- select 'Selection'
- progress to the selection_settings view, where `Include an option for ‘None of the above’` will be populated with the existing value of `is_optional` but they can edit if that's no longer correct
- get returned to the edit question page, where the is_optional checkbox no longer appears. 

The hidden input is just on this page to make passing the data around easier.